PracticeCode/AntProjects/dividerFunction1.py

```diff
@@ -0,0 +1,10 @@
+#Simple function takes two integers as arguments and divides the first int by the second int.
+#Result will include remainders if any.
+
+
+def divider (number1, number2):
+    # Func takes 2 ints as arguments.
+    # Return the result of number1 divided by number2.

```

Comment:
update comment like [this](https://github.com/python/cpython/blob/master/Lib/email/header.py#L61-L74)